doc/changes.md

```diff
@@ -1,4 +1,133 @@
+* Use colorblind-friendly colors in the scatter plots gallery example ([#1013](https://github.com/GenericMappingTools/pygmt/pull/1013))
+* Add a tutorial for plotting vectors ([#1070](https://github.com/GenericMappingTools/pygmt/pull/1070))
+* Clarify interplay of spacing and per_column in info ([#1127](https://github.com/GenericMappingTools/pygmt/pull/1127))
+* Update the check_figures_equal testing section in CONTRIBUTING.md  ([#1108](https://github.com/GenericMappingTools/pygmt/pull/1108))
+
+### Maintenance
+
+* Fix rose tests for GMT 6.2.0 ([#1324](https://github.com/GenericMappingTools/pygmt/pull/1324))
+* Add a workflow to upload baseline images as a release asset ([#1317](https://github.com/GenericMappingTools/pygmt/pull/1317))
+* Bump the minimum required GMT version to 6.2.0 ([#1321](https://github.com/GenericMappingTools/pygmt/pull/1321))
+* Add regression test for grdimage plotting an xarray.DataArray grid subset ([#1314](https://github.com/GenericMappingTools/pygmt/pull/1314))
+* Add download_test_data to download data files used in tests ([#1310](https://github.com/GenericMappingTools/pygmt/pull/1310))
+* Update fig.basemap, fig.subplot and fig.wiggle baseline images for GMT 6.2.0rc2 ([#1291](https://github.com/GenericMappingTools/pygmt/pull/1291))
+* Remove xfails and workarounds for datetime inputs into pygmt.info ([#1236](https://github.com/GenericMappingTools/pygmt/pull/1236))
+* Improve the DVC image diff workflow to support side-by-side comparison of modified images ([#1219](https://github.com/GenericMappingTools/pygmt/pull/1219))
+* Document the deprecation policy and add the deprecate_parameter decorator to deprecate parameters ([#1160](https://github.com/GenericMappingTools/pygmt/pull/1160))
+* Convert booleans arguments in build_arg_string, not in kwargs_to_strings ([#1125](https://github.com/GenericMappingTools/pygmt/pull/1125))
+* Create Github Action workflow for reporting DVC image diffs ([#1104](https://github.com/GenericMappingTools/pygmt/pull/1104))
+* Add new function to load fractures sample data ([#1101](https://github.com/GenericMappingTools/pygmt/pull/1101))
+* Update "GMT Dev Tests" workflow to test macOS-11.0 and pre-release Python packages ([#1105](https://github.com/GenericMappingTools/pygmt/pull/1105))
+* Initialize data version control for managing test images ([#1036](https://github.com/GenericMappingTools/pygmt/pull/1036))
+* Separate workflows for running tests and building documentation ([#1033](https://github.com/GenericMappingTools/pygmt/pull/1033))
+
+### Contributors
+
+* [Dongdong Tian](https://github.com/seisman)
+* [Wei Ji Leong](https://github.com/weiji14)
+* [Michael Grund](https://github.com/michaelgrund)
+* [Meghan Jones](https://github.com/meghanrjones)
+* [Will Schlitzer](https://github.com/willschlitzer)
+* [Jiayuan Yao](https://github.com/core-man)
+* [Abhishek Anant](https://github.com/itsabhianant)
+* [Michael Neumann](https://github.com/MichaeINeumann)
```

Comment:
> I did the contributors section based on the git log, but I would like to add the three others who were involved in #1070 and #1193 since this method does not recognize team programming. I will do this early tomorrow if there are no objections.

Yes! Please add them in, we need to find a way to count co-authors too. I had a brief search at stackoverflow but there doesn't seem to be a good solution somehow. Closest thing I found was https://stackoverflow.com/questions/61263083/how-to-query-git-log-for-co-authored-by/61264586#61264586 but using `git log --format="%(trailers:key=Co-authored-by)"` doesn't give the counts :sweat_smile:

```suggestion
* [Claire Klima](https://github.com/cklima616)
* [Megan Munzek](https://github.com/munzekm)
* [Michael Neumann](https://github.com/MichaeINeumann)
* [Nathan Loria](https://github.com/Nathandloria)
```